src/ui/mod.rs

```diff
@@ -141,6 +143,32 @@ impl GraphUI {
                 })
             });
     }
+
+    fn controls_window(&mut self, ctx: &egui::CtxRef, _ui: &mut egui::Ui) {
+        egui::Window::new("Controls")
+            .open(&mut self.show_controls)
+            .resizable(false)
+            .show(ctx, |ui| {
+                egui::Grid::new("theme_grid").show(ui, |ui| {
+                    ui.label("Left Click + Drag");
+                    ui.label("Move nodes, create links between nodes");
+                    ui.end_row();
+
+                    ui.label("Alt + Left Click + Drag");
+                    ui.label("Remove Links");
+                    ui.end_row();
+
+                    ui.label("Middle Mouse + Drag");
+                    ui.label("Pan the graph");
+                    ui.end_row();
+
+                    ui.label("ctrl");
```

Comment:
Minor nitpick, but can you please change "ctrl" to "Ctrl", to be consistent with the rest of the code.

Reply:
Since you merged this already I added the change in #4

Reply:
I didn't want to make a whole pr just for a typo so I bundled it up with a cargo fmt and a clippy lint, I hope that's okay